src/sentry/conf/server.py

```diff
@@ -1878,8 +1879,9 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
     "org:integrations": {"org:integrations"},
     "org:ci": {"org:ci"},
     "member:read": {"member:read"},
-    "member:write": {"member:read", "member:write"},
-    "member:admin": {"member:read", "member:write", "member:admin"},
+    "member:invite": {"member:read", "member:invite"},
```

Comment:
hm, I think this is kind of write and it doesn't make sense to have invite without read. It would mean you can invite someone but you can never know if they accepted the invite, cuz you can't read the members :/

Reply:
Ok that makes sense, fine to leave as is